tests/utils.py

```diff
@@ -155,7 +155,11 @@ def run(command: Sequence[str]) -> None:
                 f"non-zero exit status {p.returncode}\n{p.stderr.decode('utf-8')}",
             )
 
-    gnatprove = ["gnatprove", "-Ptest", "--memcached-server=localhost:11211"]
+    memcached = os.getenv("MEMCACHED")
+    if memcached:
+        gnatprove = ["gnatprove", "-Ptest", f"--memcached-server={memcached}"]
+    else:
+        gnatprove = ["gnatprove", "-Ptest"]
```

Comment:
Reading the environment variable should not be necessary, as the tests also use `defaults.gpr` (the file is copied in `_create_files`).

Reply:
Fixed.